go/oasis-node/cmd/root.go

```diff
@@ -21,10 +21,13 @@ import (
 )
 
 var (
+	// versionBuild stores current git HEAD and is set by the linker.
+	versionBuild = ""
```

Comment:
Should this live in `go/common/version/version.go` instead as we keep all the other versions there?